_includes/compose-eol.md

```diff
@@ -0,0 +1,6 @@
+> **Important**
+>
+> From April 2023 Compose V1 won't be supported anymore and will be removed from all Docker Desktop versions.
```

Comment:
@hyu don't we change the eol date to end of june?

Reply:
Silly me, this is my bad. Yes. EOL date has changed to June, I forgot to update